### PR TITLE
[fix](compile) fix be compile error

### PR DESCRIPTION
**Pull request by liaoxin01**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by liaoxin01** [comment]
run buildall

**Review by zhannngchen — DISMISSED** [review]

**Review by kaijchen — APPROVED** [review]
LGTM

**Review by Yukang-Lian — APPROVED** [review]
LGTM

File: be/src/olap/olap_meta.cpp [hunk 1]
```diff
@@ -229,7 +229,7 @@ Status OlapMeta::put(rocksdb::WriteBatch* batch) {
 
     if (!s.ok()) {
         LOG(WARNING) << "rocks db put batch failed, reason:" << s.ToString();
```
Comment (BiteTheDDDDt) [inline be/src/olap/olap_meta.cpp:231]: remove this log warning, constructor of Status will print all message.

**Comment by liaoxin01** [comment]
run buildall

File: be/src/olap/olap_meta.cpp (hunk 1 above)
Comment (liaoxin01) [reply be/src/olap/olap_meta.cpp]: done

**Review by BiteTheDDDDt — APPROVED** [review]
LGTM

**Review by luozenglin — APPROVED** [review]
LGTM

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 50.76 seconds
 stream load tsv:          506 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.9 seconds inserted 10000000 Rows, about 346K ops/s
 storage size: 17170320618 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230712202424_clickbench_pr_177258.html

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by zy-kkk — APPROVED** [review]

**Pull request merged**